Sources/ArcGISToolkit/Components/FeatureFormView/FeatureFormView.swift

```diff
@@ -71,16 +71,20 @@ public struct FeatureFormView: View {
     ///   - featureForm: The feature form defining the editing experience.
     public init(featureForm: FeatureForm) {
         _model = StateObject(wrappedValue: FormViewModel(featureForm: featureForm))
+        title = featureForm.title
```

Comment:
Initializing a state property within a view's initialize is advised against:

> "always initialize state by providing a default value in the state’s declaration" Source - [Apple](https://developer.apple.com/documentation/swiftui/state#:~:text=always%20initialize%20state%20by%20providing%20a%20default%20value%20in%20the%20state%E2%80%99s%20declaration)

Instead, I would mark the new `title` property nullable and let first run of the new `onChangeOfTitle` modifier take care of setting the initial value.

Reply:
Instead of nullable, I just set it to the empty string `@State private var title: String = ""`